### ISSUE TITLE
No .spec file in tarball as indicated by INSTALL file

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11243]
|<img alt="bjorn's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| bjorn reported|
|-|-|
|Trac ID|trac#969|
|Type|defect|
|Created|2005-04-08 17:43:24Z|

<details><summary>Searchable metadata</summary>

```
trac-id__969 969
type__defect defect
reporter__bjorn bjorn
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1112982204000000 1112982204000000
changetime__1143085997000000 1143085997000000
version__ 
owner__ 
cc__radix cc__exarkun cc__bjorn
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="bjorn's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|bjorn commented|
|-|-|

```
#!html
<pre>
The INSTALL file in Twisted-2.0.0.tar.bz2 says "For RPM based platforms there is
a spec file in the admin/ folder in the tarball." The tarball does not, however,
contain a admin folder let alone a spec file.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
Hooray, release bug type thing.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set status to `closed`|
|-|-|

sorry, the INSTALL file was out of date. Fixed in #16359.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
